io_export_paper_model.py

```diff
@@ -1179,6 +1186,94 @@ def __init__(self, uvedge, size, index):
         self.bounds = [self.center, self.center + (1.2 * normal + tangent) * size, self.center + (1.2 * normal - tangent) * size]
 
 
+# /////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+the_path = "Stickers/tooth-pulltab.svg"
+logger = logging.getLogger(__name__)
+ns = {"u": "http://www.w3.org/2000/svg"}
+
+vertices = []
+
+def load_svg(path):
+    parser = etree.XMLParser(remove_comments=True, recover=True)
+    try:
+        doc = etree.parse(path, parser=parser)
+        svg_root = doc.getroot()
+    except Exception as exc:
+        logger.error("Failed to load input file! (%s)" % exc)
+    else:
+        return svg_root
+
+
+def svg2uv(path):
+    # vertices.clear()
+
+    svg_root = load_svg(path)
+    if svg_root is None:
+        print("SVG import blowed up, no root!")
+        return
+
+    polylines = svg_root.findall("u:polyline", ns)
+    paths = svg_root.findall("u:path", ns)
+
+    # Make Polyline Vectors
+    polyline_vectors = []
+
+    for p in polylines:
+        points = p.attrib['points']
+        polyline_vectors += vectorize_polylines(points)
+    for v in polyline_vectors:
+        makeUVVertices(v)
+        print("p")
+    # Make Path vectors
+    path_vectors = []
+    for p in paths:
+        path = p.attrib['d']
+        path_vectors += vectorize_paths(path)
+    return vertices
+
+
+
+def vectorize_paths(path):
+    # "M0,0H250V395.28a104.71,104.71,0,0,0,11.06,46.83h0A104.71,104.71,0,0,0,354.72,500h40.56a104.71,104.71,0,0,0,93.66-57.89h0A104.71,104.71,0,0,0,500,395.28V0"
+    r = re.compile('[MmHhAaVv][\d,\.-]*')  # split by commands
+    p = re.sub(r'-', r',-', path)  # make sure to catch negatives
+    commands = r.findall(p)
+    for c in commands:
+        command = c[0]
+        parameters = [float(i) for i in c[1:].split(",")]
+        print(command, parameters)
+
+    print(commands)
+    return []
+
+
+def vectorize_polylines(points):
+    points = points.replace(",", " ") 
+    ps = points.split()
+    xs = ps[0::2]  # every second element starting at 0
+    ys = ps[1::2]  # every second element starting at 1
+    lines = []
+
+    for i in range(0, len(xs)):
+        x1 = float(xs[i])
+        y1 = float(ys[i])
+        # fs = [float(i) for i in [x1,y1,x2,y2]]
+        o = {'x1': x1, 'y1': y1}
+        lines.append(o)
+    return lines
+
+
+def makeUVVertices(v):
+    v1 = UVVertex(M.Vector((v["x1"], v["y1"])) * 0.00001) #scaling down to avoid overflow
```

Comment:
what overflow problem is it? maybe clamp instead?

Reply:
I think it has to do with SVG size - in blender it complains that it won't fit on the page in its original size